client/src/GlobalStyle.js

```diff
@@ -21,22 +21,42 @@ const GlobalStyle = createGlobalStyle`
     --info-text-color:#565959;
     --font-size: 16px;
 }
+html{
+  margin: 0;
+  padding: 0;
+}
 body
  {
   margin: 0;
+  padding:0;
   font-size: 16px;
   font-family: "Trebuchet", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   color: var(--info-text-color)
 
 }
+main{
+  margin: 0;
+  padding: 0;
+}
```

Comment:
its not necessary to stuck that in all three sections, you could put into the * section if it should be applied to everything and you want to decide everything by yourself
```suggestion
main{
}
```

Reply:
Thanks! You're right!